db/seeds.rb

```diff
@@ -1,39 +1,42 @@
-FeedSource.create(feed_type: 'general', name: 'Crypto Insider', feed_url: 'https://cryptoinsider.com/content/feed/', site_url: 'https://cryptoinsider.com/')
-FeedSource.create(feed_type: 'general', name: 'Crypto Briefing', feed_url: 'https://cryptobriefing.com/feed/', site_url: 'https://cryptobriefing.com/')
-FeedSource.create(feed_type: 'general', name: 'Crypto Daily', feed_url: 'https://cryptodaily.co.uk/feed/', site_url: 'https://cryptodaily.co.uk/')
-FeedSource.create(feed_type: 'general', name: 'CryptoNinjas', feed_url: 'https://www.cryptoninjas.net/feed/', site_url: 'https://www.cryptoninjas.net/')
-FeedSource.create(feed_type: 'general', name: 'CryptoPost', feed_url: 'http://cryptopost.com/feed/', site_url: 'http://cryptopost.com/')
-FeedSource.create(feed_type: 'general', name: 'Cryptovest', feed_url: 'https://cryptovest.com/feed/', site_url: 'https://cryptovest.com/')
-FeedSource.create(feed_type: 'general', name: 'ETHNews', feed_url: 'https://www.ethnews.com/rss.xml', site_url: 'https://www.ethnews.com/')
-FeedSource.create(feed_type: 'general', name: 'Ethereum World News', feed_url: 'https://ethereumworldnews.com/feed/', site_url: 'https://ethereumworldnews.com/')
-FeedSource.create(feed_type: 'general', name: 'ForkLog', feed_url: 'http://forklog.net/feed/', site_url: 'http://forklog.net/')
-FeedSource.create(feed_type: 'general', name: 'Howtotoken', feed_url: 'https://howtotoken.com/feed/', site_url: 'https://howtotoken.com/')
-FeedSource.create(feed_type: 'general', name: 'Invest In Blockchain', feed_url: 'https://www.investinblockchain.com/feed/', site_url: 'https://www.investinblockchain.com/')
-FeedSource.create(feed_type: 'general', name: 'Live Bitcoin News', feed_url: 'http://www.livebitcoinnews.com/feed/', site_url: 'http://www.livebitcoinnews.com/')
-FeedSource.create(feed_type: 'general', name: 'NewsBTC', feed_url: 'https://www.newsbtc.com/feed/', site_url: 'https://www.newsbtc.com/')
-FeedSource.create(feed_type: 'general', name: 'Profit Confidential', feed_url: 'https://www.profitconfidential.com/feed/', site_url: 'https://www.profitconfidential.com/')
-FeedSource.create(feed_type: 'general', name: 'The Bitcoin News', feed_url: 'https://thebitcoinnews.com/feed/', site_url: 'https://thebitcoinnews.com/')
-FeedSource.create(feed_type: 'general', name: 'The Daily Hodl', feed_url: 'https://dailyhodl.com/feed/', site_url: 'https://dailyhodl.com/')
-FeedSource.create(feed_type: 'general', name: 'Trustnodes', feed_url: 'https://www.trustnodes.com/feed', site_url: 'https://www.trustnodes.com/')
-FeedSource.create(feed_type: 'general', name: 'UseTheBitcoin', feed_url: 'https://usethebitcoin.com/feed/', site_url: 'https://usethebitcoin.com/')
+FeedSource.find_or_create_by(feed_type: 'general', name: 'AMBCrypto', feed_url: 'https://ambcrypto.com/feed/', site_url: 'https://ambcrypto.com/')
```

Comment:
Hm, IIRC `FeedSource.create` would just fail silently if it already exists; any reason for changing this to `find_or_create_by`?

Reply:
Create was failing just not silently.   It was throwing errors like:
PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_feed_sources_on_name"
DETAIL:  Key (name)=(AMBCrypto) already exists.

So changed to find_or_create_by to stop those.